docs/deploying.md

```diff
@@ -89,6 +89,8 @@ docker run -d -p 5000:5000 \
 	registry:2
 ```
 
+If the certificate issuer supplies you with an 'intermediate' certificate, you need to combine your certificate with the intermediates to form a 'certificate bundle'. You can do this using the cat command: ```cat server.crt intermediate-certificates.pem > server.with-intermediate.crt```. You can then configure the registry to use your certificate bundle with the ```REGISTRY_HTTP_TLS_CERTIFICATE``` environment variable.
```

Comment:
@rxbynerd thank you for the contribution.  There are a few Markdown issues and minor punctuation.  Also, I think some sentence structures can be improved;

--> https://gist.github.com/moxiegirl/d50d441aabc8b32d4e5e

A certificate issuer may supply you with an *intermediate* certificate. In this case, you must combine your certificate with the intermediate's to form a *certificate bundle*. You can do this using the `cat` command: 

```
$ cat server.crt intermediate-certificates.pem > server.with-intermediate.crt
```

You then configure the registry to use your certificate bundle using the `REGISTRY_HTTP_TLS_CERTIFICATE` environment variable.